lib/streamlit/type_util.py

```diff
@@ -368,6 +370,14 @@ def is_altair_chart(obj: object) -> bool:
     return is_type(obj, _ALTAIR_RE)
 
 
+_PILLOW_RE: Final = re.compile(r"^PIL\..*")
+
+
+def is_pillow_image(obj: object) -> bool:
```

Comment:
@LukasMasuch do we need this function as part of this PR?

Reply:
Ahh, good catch. Not really, for this specific PR. But for another lazy-loading PR.

Reply:
Will merge with this method for now, since I anyways will wrap up the other PR that needs it today. And it our CI doesn't work right now I would do another commit.

Reply:
https://github.com/streamlit/streamlit/pull/8134